airflow/hooks/ecs_hook.py

```diff
@@ -0,0 +1,26 @@
+from airflow.hooks.base_hook import BaseHook
+from airflow.utils import AirflowException
+import json
+import boto3
+
+class ECSHook(BaseHook):
+    
+    def __init__( self, aws_conn_id="aws_credentials"):
+        self.conn = self.get_connection(aws_conn_id)    
+        self._a_key = None
+        self._s_key = None
+        self.extra_params = json.loads(self.conn.extra)
+        if 'aws_secret_access_key' in self.extra_params:
```

Comment:
This prevents boto3 from transparently picking up IAM roles available via the instance's instance profile.